bazel_defaults

```diff
@@ -66,23 +66,22 @@ build:gdr --define=with_gdr_support=true
 build:ngraph --define=with_ngraph_support=true
 build:verbs --define=with_verbs_support=true
 
+
 build --define=use_fast_cpp_protos=true
 build --define=allow_oversize_protos=true
 build --define=grpc_no_ares=true
 
-build --spawn_strategy=standalone
-build --genrule_strategy=standalone
+
+#build --spawn_strategy=standalone
+#build --genrule_strategy=standalone
 build -c opt
+# Default paths for TF_SYSTEM_LIBS
+#build --define=PREFIX=/usr
+#build --define=LIBDIR=$(PREFIX)/lib
+#build --define=INCLUDEDIR=$(PREFIX)/include
```

Comment:
Don't comment these either, they'll break system libs stuff. The ./configure script is already setup to handle overrides for these. These are needed as the default fallback for when the user chooses not to override them.